rules/windows/file/file_event/file_event_win_screenconnect_remote_tool_execution.yaml

```diff
@@ -0,0 +1,25 @@
+title: Remote Access Tool - ScreenConnect Remote Tool Execution
+id: 0afecb6e-6223-4a82-99fb-bf5b981e92a5
+related:
+    - id: b1f73849-6329-4069-bc8f-78a604bb8b23
+      type: similar
+status: experimental
+description: |
+    Detects the creation of files in a specific location by ScreenConnect RMM.
+    ScreenConnect has feature to remotely execute binaries on a target machine. These binaries will be dropped to ":\Users\User\Documents\ConnectWiseControl\Temp\" before execution.
+author: Ali Alwashali
+date: 2023/10/10
+tags:
+    - attack.execution
+    - attack.t1059.003
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\ScreenConnect.WindowsClient.exe'
+        TargetFilename|contains: '\Documents\ConnectWiseControl\Temp\'
```

Comment:
Is this location fixed or can be changed?

Reply:
I think it's fixed, I didn't find an option to change it